packages/core/src/makeGeneralUtils.js

```diff
@@ -1,5 +1,20 @@
 module.exports = makeGeneralUtils
 
+function patchDocumentAccess(doc, node) {
+  // apply patch only to DOM nodes
+  const isInstanceOfNode = global?.window?.Node && node instanceof global?.window?.Node
+  if (!isInstanceOfNode) {
+    return
+  }
+  // apply patch only once
+  const isAlreadyPatched = node.hasOwnProperty('ownerDocument')
+  if (isAlreadyPatched) {
+    return
+  }
+  // when ownerDocument prop is being accessed return proxy document instead of real document
+  Object.defineProperty(node, 'ownerDocument', { get: () => doc })
```

Comment:
without `writable: false` can I simply 
```js
delete node.ownerDocument
```
and go bask to accessing it via the prototype?